packages/components/pie-modal/src/index.ts

```diff
@@ -119,8 +112,11 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
         super.disconnectedCallback();
     }
 
-    firstUpdated (changedProperties: DependentMap<ModalProps>) : void {
+    async firstUpdated (changedProperties: PropertyValues<this>) : Promise<void> {
+        super.firstUpdated(changedProperties);
```

Comment:
just curious, why we had to call super.firstUpdated here?
same goes for         super.updated(changedProperties);

Reply:
I am unsure if the order of the super call is important within the block. However, my understanding and experience is that not calling the base class super method leads to weird behaviours with lifecycle methods. It's the same reason you must always call the super constructor.